src/components/Dashboard/Welcome/Welcome.tsx

```diff
@@ -15,18 +15,12 @@ const Welcome = ({ firstName }: Props): ReactElement => {
   const { appName } = useGetAppSettings();
   const currentHour = today.hour;
 
-  let greeting = firstName
-    ? t('Good Evening, {{ firstName }}.', { firstName })
-    : t('Good Evening,');
+  let greeting = t('Good Evening,') + (firstName ? ` ${firstName}.` : '');
 
   if (currentHour < 12) {
-    greeting = firstName
-      ? t('Good Morning, {{ firstName }}.', { firstName })
-      : t('Good Morning,');
+    greeting = t('Good Morning,') + (firstName ? ` ${firstName}.` : '');
   } else if (currentHour < 18) {
-    greeting = firstName
-      ? t('Good Afternoon, {{ firstName }}.', { firstName })
-      : t('Good Afternoon,');
+    greeting = t('Good Afternoon,') + (firstName ? ` ${firstName}.` : '');
```

Comment:
How about removing the string concatenation and after the last `else if` doing

```
if (firstName) {
  greeting += ` ${firstName}.`
}
```

Also, this seems good for now, but if find any more bugs like this, we should probably look into tweaking i18n's interpolation settings.

Reply:
I'm sure there are many more bugs because of the interpolation settings. My work around in the past has been to use a <Trans /> component where I can set `interpolation: { escapeValue: false },`